lib/migration_data/active_record/migration.rb

```diff
@@ -15,6 +15,8 @@ def self.included(base)
         base.class_eval do
           def exec_migration_with_data(conn, direction)
             origin_exec_migration(conn, direction)
+            ::ActiveRecord::Base.connection.schema_cache.clear!
+            return if Rails.env.test? && MigrationData.config.skip_data_on_test
```

Comment:
This option can be useful not only test env, but for others as well. So let's go this way:

```ruby
return if MigrationData.config.skip_data_on_test
```